test/serialize.js

```diff
@@ -61,6 +131,14 @@ test('parse->serialize', function() {
       cookie.serialize('cat', ' ";/')));
 });
 
+test('encoded method validation', function() {
+   assert.throws(cookie.serialize.bind(null, 'cat', ' ', {
+       encode: 42
+   }), /number is not a function/);
```

Comment:
I saw that earlier and threw in a regex that should cover it in either case. Let me know though.